### ISSUE TITLE
PageUrl with field uid

### ISSUE THREAD

**Issue by joeschi** [b13/masi#19]
Masi works fine except trying to include the page-uid into the url-path. I have this configuration:

```
  TCEMAIN.pages.slug.generatorOptions {
    prefixParentPageSlug = 0
    fields = title, uid
    fieldSeparator = -
    prefix = /blog
  }
```
Creating a new page the prefix works, the title also, but the uid is still not present at this moment. After the page is created recalculating the url-pathsegement adds the uid as expected. Is there any chance to get the uid at the point of creating a new page?

**Comment by fwg** [comment]
This is maybe fixed by #37... Checking if it is. 

Edit: yes, it's fixed. We use it for tx_news_domain_model_news, but have the same fields configuration.

**Comment by bmack** [comment]
Thanks for the info @fwg
